docker-compose-hw.yaml

```diff
@@ -0,0 +1,118 @@
+# Copyright 2018 Intel Corporation
```

Comment:
Rename the fine as docker-compose-sgx-hw.yaml

Reply:
Ok